aict_tools/io.py

```diff
@@ -424,8 +424,7 @@ def read_telescope_data(
             first=first,
             last=last,
         )
-        if aict_config.disp:
-            df = convert_units(df, aict_config.disp)
+        df = convert_units(df, aict_config.disp)
```

Comment:
This will only work if there is a disp config defined

Reply:
already on a fix